src/widgets/goalStatusGraph.js

```diff
@@ -0,0 +1,40 @@
+import { Op } from 'sequelize';
+import {
+  Goal, Grant, Recipient, sequelize,
+} from '../models';
+
+export default async function goalStatusGraph(scopes) {
+  const goals = await Goal.findAll({
+    where: {
+      [Op.and]: [
+        scopes.goal,
+        {
+          status: {
+            [Op.not]: null,
```

Comment:
Elsewhere, we've been using "needs status" if the goal was created in the hub and therefore doesn't have a status. Do you think we should use it here?

Reply:
"needs status" and "draft" goals are ignored in the widget. I'm updating the backend to explicitly ignore draft goals